devdocs/updatenotes.md

```diff
@@ -9,6 +9,12 @@
 * Deprecated old drivers (conflicts, etc)
     - version conflicts (removing cql drivers 1.9.* 3.*)
 
+# Performance
+* dryrun capability
```

Comment:
Caps for start of each of the performance items.

Reply:
ah, yes. I'll be removing this doc shortly!